bofire/priors/mapper.py

```diff
@@ -13,9 +13,16 @@ def map_GammaPrior(data_model: data_models.GammaPrior) -> gpytorch.priors.GammaP
     )
 
 
+def map_LKJPrior(data_model: data_models.LKJPrior) -> gpytorch.priors.LKJPrior:
```

Comment:
Can you add a test for this function to make sure that the map works correctly?

Reply:
Done.